rust/aura/src/main.rs

```diff
@@ -32,14 +32,12 @@ fn main() -> Result<(), Error> {
     // Establish common file paths.
     let logp: &Path = args
         .logfile
-        .as_ref()
-        .map(|p| p.as_path())
+        .as_deref()
```

Comment:
I will remember this.

Reply:
Oh, come on, just throw `&` and `*` at it — as many as needed, until it compiles.